### PR TITLE
create new contact ui

### PR DESCRIPTION
**Pull request by mohit-s96**
## current progress screenshot
create-contact

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Contacts/Pane/Create.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import { Pane, Typography } from "neetoui";
+
+import Form from "./Form";
+
+import {
+  CONTACTS_FORM_INITIAL_FORM_VALUES,
+  NO_OP_FUNCTION,
+} from "../constants";
+
+export default function NewContactPane({ showPane, setShowPane }) {
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/Pane/Create.jsx:12]: Please use arrow functions. Prefer using arrow functions at all times.

File: app/javascript/src/components/Dashboard/Contacts/constants.js [hunk 2]
```diff
@@ -2,6 +2,23 @@ import React from "react";
 
 import { MenuHorizontal } from "@bigbinary/neeto-icons";
 import { Avatar, Typography } from "neetoui";
+import * as yup from "yup";
+
+function noOp() {}
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/constants.js:7]: Make this an arrow function and name it `noop`

File: app/javascript/src/components/Dashboard/Contacts/constants.js [hunk 3]
```diff
@@ -2,6 +2,23 @@ import React from "react";
...
+
+export const NO_OP_FUNCTION = noOp;
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/constants.js:9]: This is unnecessary. Remove it.

**Review by yedhink — COMMENTED** [review]
@mohit-s96 Few comments.

**Comment by mohit-s96** [comment]
Thanks for the suggestions. I'll make the changes.

**Pull request merged**